cmd/eval_test.go

```diff
@@ -281,6 +281,26 @@ func TestEvalWithBundleDuplicateFileNames(t *testing.T) {
 	})
 }
 
+func TestEvalWithStrictBuiltinErrors(t *testing.T) {
+	params := newEvalCommandParams()
+	params.strictBuiltinErrors = true
+
+	var buf bytes.Buffer
+	_, err := eval([]string{"1/0"}, params, &buf)
+	if err == nil {
+		t.Fatal("expected error")
+	}
+
+	params.strictBuiltinErrors = false
+	buf.Reset()
+
+	_, err = eval([]string{"1/0"}, params, &buf)
+	if err != nil {
+		t.Fatal("unexpected error:", err)
+	}
```

Comment:
Could we add what it actually returns with `params.strictBuiltinErrors` set to `false`? (Can we assert "undefined" here?)

Reply:
Fixed.